packages/mobile/src/components/molecules/CalendarHeader/index.tsx

```diff
@@ -1,33 +1,39 @@
 import { useMemo } from "react";
 
+import classnames from "classnames";
 import ArrowButton from "src/page/Calendar/ArrowButton";
+import { DAY } from "src/utils/calendarTools";
+import { calendarZeroPad } from "src/utils/calendarZeroPad";
 
 import $ from "./style.module.scss";
 
 type Props = {
-  year: number;
-  month: number;
-  onMonthDecrease: () => void;
-  onMonthIncrease: () => void;
+  isSticky?: boolean;
+  calendar: {
+    year: number;
+    month: number;
+    date?: number;
+    day?: number;
+  };
```

Comment:
이 부분 일정 페이지에서는 year, month만 사용하고 식단 페이지에서는 전부 사용하는 걸로 알고 있는데, 유니온 타입으로 만들어서 두 상태만 가능하다는 걸 명시하면 어떨까요?

```suggestion
  calendar: {
    year: number;
    month: number;
  } | {
    year: number;
    month: number;
    date: number;
    day: number;
  };
```

Reply:
엇 한 번 사용해봤는데 
```
Property 'date' does not exist on type '{ year: number; month: number; } | { year: number; month: number; date: number; day: number; }
```
요런 오류가 뜨네요ㅠ